internal/cli/project_destroy.go

```diff
@@ -0,0 +1,100 @@
+package cli
+
+import (
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/pkg/server/gen"
+	"strings"
+)
+
+type ProjectDestroyCommand struct {
+	*baseCommand
+
+	skipDestroyResources bool
+	confirm              bool
+}
+
+func (c *ProjectDestroyCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(c.Flags()),
+	); err != nil {
+		return 1
+	}
+
+	// Get the project we're destroying
+	project, err := c.project.Client().GetProject(c.Ctx, &pb.GetProjectRequest{
+		Project: c.project.Ref(),
+	})
+	if err != nil {
+		return 1
+	}
+
+	// Confirmation is required for destroying a project &/or its resources
+	if !c.confirm {
+		proceed, err := c.ui.Input(&terminal.Input{
+			Prompt: "Do you really want to destroy project \"" + project.Project.Name + "\" and its resources? Only 'yes' will be accepted to approve: ",
+			Style:  "",
+			Secret: false,
+		})
+		if err != nil {
+			c.ui.Output(
+				"Error getting input: %s",
+				clierrors.Humanize(err),
+				terminal.WithErrorStyle(),
+			)
+		} else if strings.ToLower(proceed) != "yes" {
+			c.ui.Output("Destroying project %q and resources requires confirmation.", project.Project.Name, terminal.WithWarningStyle())
+			return 1
+		}
```

Comment:
If I read this correctly, if error is not `nil` then we output an error message but continue on and attempt to destroy things regardless if the user input `yes` or anything

Reply:
Was this resolved by mistake or am I missing something? The problem appears to still be present. Am I mistaken?